lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -132,6 +149,7 @@ def main():
             port=dict(default="5432"),
             db=dict(required=True),
             ext=dict(required=True, aliases=['name']),
+            version=dict(default="", aliases=['version']),
```

Comment:
`DOCUMENTATION` metadata states that default value for `version` is `None`, not an empty string. Documentation and code must match.